packages/core/src/components/essentials/tooltip.ts

```diff
@@ -207,10 +208,26 @@ export class Tooltip extends Component {
 	positionTooltip(e: CustomEvent) {
 		const holder = this.services.domUtils.getHolder();
 		const target = this.tooltip.node();
+		const isZoomBarEnabled = Tools.getProperty(
+			this.model.getOptions(),
+			"zoomBar",
+			"top",
+			"enabled"
+		);
 
 		let mouseRelativePos = Tools.getProperty(e, "detail", "mousePosition");
 		if (!mouseRelativePos) {
 			mouseRelativePos = mouse(holder);
+		} else {
+			// if the mouse position is from event (ruler)
+			// we need add zoom bar height
+			if (isZoomBarEnabled) {
```

Comment:
what if zoom bar gets positioned in the bottom of the chart?

Reply:
> what if zoom bar gets positioned in the bottom of the chart?
The `isZoomBarEnabled` is actually checking the top zoom bar.
So it won't change y position if zoom bar is in the bottom of the chart.